### PR TITLE
Added Initial scraper which scrapes all the links from a given URL

### PR DESCRIPTION
**Pull request by daminisatya**

### CODE REVIEW

**Comment by shivenmian** [comment]
Please add jsoup to build.gradle as well, build is failing because of it.

**Comment by daminisatya** [comment]
@shivenmian Added. Thanks!

**Comment by shivenmian** [comment]
@daminisatya Build will still fair. Remove .jar from the added line. It should just be ```compile name: 'jsoup-1.9.2'``` :D

**Comment by daminisatya** [comment]
Oops. My bad. (Y)

**Comment by shivenmian** [comment]
Looks good to me.

File: src/org/loklak/api/search/GenericScraper.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
+/**
+ *  GenericScraper
+ *  Copyright 22.02.2015 by Damini Satya, @daminisatya
```
Comment (jig08) [inline src/org/loklak/api/search/GenericScraper.java]: @daminisatya Erroneous date, I suppose :sweat_smile:

Comment (daminisatya) [inline src/org/loklak/api/search/GenericScraper.java]: Oh yeah. Took the copyright stuff from other file. Thanks for pointing out.

Comment (jig08) [inline src/org/loklak/api/search/GenericScraper.java]: Yes, I figured that :)

**Pull request merged**